Modules/_ctypes/_ctypes.c

```diff
@@ -3483,7 +3483,6 @@ PyCFuncPtr_FromDll(PyTypeObject *type, PyObject *args, PyObject *kwds)
         return NULL;
     }
 #endif
-    Py_INCREF(dll); /* for KeepRef */
```

Comment:
`dll` is a borrowed reference owned by `ftuple`. It should be increfed before decrefing `ftuple`.

Reply:
You're absolutely right.